### PR TITLE
rtmp-services: Add Volume.com

### PR DESCRIPTION
**Pull request by sergey-mm**
### Description
Add https://volume.com servers to rtmp-services.

### Motivation and Context
Adding support to the volume.com streaming platform

### How Has This Been Tested?
Local build

### Types of changes
New feature (non-breaking change which adds functionality)

### Checklist:

- [x] My code has been run through [clang-format](https://github.com/obsproject/obs-studio/blob/master/.clang-format).
- [x] I have read the [**contributing** document](https://github.com/obsproject/obs-studio/blob/master/CONTRIBUTING.rst).
- [x] My code is not on the master branch.
- [x] The code has been tested.
- [x] All commit messages are properly formatted and commits squashed where appropriate.
- [ ] I have included updates to all appropriate documentation.

### CODE REVIEW

File: plugins/rtmp-services/data/services.json [hunk 1]
```diff
@@ -2193,6 +2193,37 @@
                 "max audio bitrate": 320,
                 "x264opts": "scenecut=0"
             }
+        },
+        {
+            "name": "Volume",
+	    "alt_names": ["Volume.com"],
```
Comment (derrod) [inline plugins/rtmp-services/data/services.json:2199]: This is only required when a service is being migrated, as this is a new service this line does not need to be here (also the indentation is wrong).

**Comment by derrod** [comment]
We require PRs for new services to be made by representatives of the service, is that the case ehre?

**Comment by sergey-mm** [comment]
@derrod, yes, I'm a developer of the service

File: UI/window-basic-settings-stream.cpp [hunk 2]
```diff
@@ -298,6 +298,8 @@ void OBSBasicSettings::UpdateKeyLink()
 			"https://www.openrec.tv/login?keep_login=true&url=https://www.openrec.tv/dashboard/live?from=obs";
 	} else if (serviceName == "Brime Live") {
 		streamKeyLink = "https://brimelive.com/obs-stream-key-link";
+	} else if (serviceName == "Volume") {
```
Comment (bershanskiy) [inline UI/window-basic-settings-stream.cpp:301]: This probably should be "Volume.com" to match the JSON data.

**Comment by sergey-mm** [comment]
Hey, @derrod does this PR have any blocker?

**Comment by derrod** [comment]
No real blockers from your side, we're currently evaluating #4737 as an option that does not require a new piece of code to be added for each service to make service PRs a little cleaner. I'll hold off on merging until that is resolved.

**Comment by sergey-mm** [comment]
@derrod @jp9000  is it any blocker for this PR? Looks like [another](https://github.com/obsproject/obs-studio/pull/4847) service was recently added but https://github.com/obsproject/obs-studio/pull/4737 still not merged

**Comment by Fenrirthviti** [comment]
That PR probably shouldn't have been merged. Jim sometimes jumps the gun like that :)

**Comment by jp9000** [comment]
Manually merged in ead931ed17eff603058bcd799b35dd49f9d4fa7e

**Pull request closed without merging**